srcpkgs/tau/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'tau'
+pkgname=tau
+version=0.9.2
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo pkg-config rust glib-devel"
+makedepends="gtk+3-devel rust-std"
+checkdepends="appstream-glib desktop-file-utils"
+short_desc="GTK frontend for the Xi text editor, written in Rust"
+maintainer="Enno Boland <gottox@voidlinux.org>"
```

Comment:
Why are you not the maintainer?

@Gottox weigh in, I suppose GXI was your package so you may have more knowledge about this package.

Reply:
Ah, I guess thie was changed when Gottox adopted all of my packages. I can take maintainership of this again if you want to.

Reply:
Why did that initial adoption take place?

Reply:
Because I'm not really active on Void anymore, so Gottox asked if he can take over my packages which was OK with me

Reply:
then I'm OK with this maintainership.